### ISSUE TITLE
Remove from context button lacks aria label

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#236094]
<img width="198" alt="Image" src="https://github.com/user-attachments/assets/87bcd314-8160-498e-aff5-934037715012" />
cc @jooyoungseo
